src/invidious/views/watch.ecr

```diff
@@ -35,11 +35,11 @@
 <h1>
     <%= HTML.escape(video.title) %> 
     <% if params[:listen] %>
-    <a href="/watch?<%= env.params.query %>&listen=0">
+    <a title="<%=translate(locale, "Video mode")%>" href="/watch?<%= env.params.query %>&listen=0">
         <i class="icon ion-ios-videocam"></i>
     </a>
     <% else %>
-    <a href="/watch?<%= env.params.query %>&listen=1">
+    <a title="<%=translate(locale, "Audio only mode")%>" href="/watch?<%= env.params.query %>&listen=1">
```

Comment:
This should be "Audio mode" or the locale should be "Audio only mode". Personally prefer "Audio mode" but whichever you prefer is fine as long as it's consistent.

Reply:
I didn't notice that before sending the commit, I thought I had left it as "Audio mode" both in the locale and in watch.ecr, if you correct that detail for me it would be fine.